.github/workflows/pull-requests.yml

```diff
@@ -6,15 +6,15 @@ on:
 
 jobs:
   build_python_artifacts:
-    name: Build Python Artifacts
-    uses: jondavid-black/AaC/.github/workflows/python-build-artifacts.yml@923464a8542c6cfa46018b45f50f1ffc455d5de9
+    name: Python Build
+    uses: jondavid-black/AaC/.github/workflows/python-build-artifacts.yml@5f36be5cb244cad6aaa226fb6cbae07a63ad9074
 
   build_vscode_artifacts:
-    name: Build VSCode Extension Artifacts
+    name: VSCode Ext. Build
```

Comment:
```suggestion
    name: VSCode Extension Build
```

Reply:
Spelling out Extension makes the step name unreadable in Github's interface. ☹️